src/Validation/Validation.php

```diff
@@ -1231,7 +1231,7 @@ public static function imageSize($file, $options)
         if (isset($options['width'])) {
             $validWidth = self::comparison($width, $options['width'][0], $options['width'][1]);
         }
-        if (isset($validHeight) && isset($validWidth)) {
+        if (isset($validHeight, $validWidth)) {
```

Comment:
This should actually be `!== null` both - using isset() with existing vars is a big code smell / antipattern.